components/encumbered/lms/lms.p5m

```diff
@@ -0,0 +1,23 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2018 <contributor>
+#
+
+<include lms.proto-objects>
```

Comment:
Why do you split proto objects to separate manifest? It's not typical and confusing.

Reply:
Well, I thought it would be more "automated" to have the stuff coming from sample-manifest in one file as proto.objects beiing included in the more "static" customized, handmade-part containing base p5m.

But I changed it.